lib/appium/AppiumRunner.js

```diff
@@ -102,6 +102,24 @@ function getConfigPath(appPath) {
     return path.join(appPath, 'config.xml');
 }
 
+function installAppiumServer() {
+    /*
+     * We install appium in this counter-intuitive way at runtime in order to
+     * not trigger massive dependency downloads via `npm install` for autotest
+     * execution. Installing appium only happens in the local-running case, and
+     * thus is not needed in our CI system. By using this method, we save the CI
+     * a lot of time.
+     * ALTERNATIVE: what if paramedic dependencies were installed via `npm install
+     * --production`, and appium was stuffed into `devDependencies`. That should
+     * also prevent this problem? 
```

Comment:
That is certainly a viable option, but I'm still reluctant to do so because paramedic is used not only in CI, but also by people who want to test things quickly and they would certainly run `npm i` and not `npm i --production` and end up with extra 5 minutes of installation and 250MB of dependencies which are not used in most cases. Not to mention various npm installation issues...